### ISSUE TITLE
Log created for an outdated container from each watcher 

### ISSUE THREAD

**Issue by mwolter805** [fmartinou/whats-up-docker#64]
Hello,
As noted in the HA forums [here](https://community.home-assistant.io/t/whats-up-docker-how-to-keep-your-containers-up-to-date/168315/137?u=mwolter), a log for a single outdated container is sent from each watcher. Even when the container is Not sure if this is affecting triggers as I do not have any.
![Screen Shot 2021-09-12 at 8 44 01 AM](https://user-images.githubusercontent.com/24851651/132993983-0e4e2623-2d96-4fbb-932a-07ab5ca02c13.jpg)

Attached is the full docker run command as well as the logs for the hour. 
[WUD Logs.txt](https://github.com/fmartinou/whats-up-docker/files/7149857/WUD.Logs.txt)
[WUD Docker.txt](https://github.com/fmartinou/whats-up-docker/files/7149858/WUD.Docker.txt)

**Comment by fmartinou** [comment]
Hi @mwolter805 ,

Could you give a try to `fmartinou/whats-up-docker:fix__64`?

It should fix your issue 🤞 .

**Comment by mwolter805** [comment]
Looks much better and the containers are with the correct host. Thanks!
```
12:06:15.255  INFO whats-up-docker/watcher.docker.lokilxc: Cron finished (1 containers, 0 updates available)
12:06:11.416  INFO whats-up-docker/watcher.docker.homeassistantlxc: Cron finished (2 containers, 0 updates available)
12:06:10.818  INFO whats-up-docker/watcher.docker.gatewaypi: Cron finished (6 containers, 0 updates available)
12:06:10.606  INFO whats-up-docker/watcher.docker.noderedlxc: Cron finished (1 containers, 0 updates available)
12:06:10.591  INFO whats-up-docker/watcher.docker.prometheuslxc: Cron finished (3 containers, 2 updates available)
12:06:10.583  INFO whats-up-docker/watcher.docker.grafanalxc: Cron finished (2 containers, 0 updates available)
12:06:10.562  INFO whats-up-docker/watcher.docker.timescaledblxc: Cron finished (4 containers, 0 updates available)
12:06:10.548  INFO whats-up-docker/watcher.docker.frigatelxc: Cron finished (1 containers, 0 updates available)
12:06:10.530  INFO whats-up-docker/watcher.docker.utilitylxc: Cron finished (4 containers, 1 updates available)
12:06:10.526  INFO whats-up-docker/watcher.docker.utilitylxc: Update available (container=utilitylxc_esphome)
12:06:10.522  INFO whats-up-docker/watcher.docker.prometheuslxc: Update available (container=prometheuslxc_alert-manager)
12:06:10.508  INFO whats-up-docker/watcher.docker.prometheuslxc: Update available (container=prometheuslxc_prometheus)
12:06:10.494  INFO whats-up-docker/watcher.docker.piholelxc: Cron finished (1 containers, 1 updates available)
12:06:10.494  INFO whats-up-docker/watcher.docker.piholelxc: Update available (container=piholelxc_pihole)
```

**Comment by fmartinou** [comment]
Fix available in version `5.5.0`

**Comment by mwolter805** [comment]
Installed, will let you know. Thanks!
